modules/http4s/src/main/scala/pureconfig/module/http4s/package.scala

```diff
@@ -1,8 +1,8 @@
 package pureconfig.module
 
 import org.http4s.Uri
-import pureconfig.ConfigReader
 import pureconfig.error.CannotConvert
+import pureconfig.{ConfigReader, ConfigWriter}
```

Comment:
The build is failing because this line doesn't conform to the Scalariform rule (running `sbt compile` should fix this automatically). Could you please fix this?